src/components/Vocal.js

```diff
@@ -171,8 +210,8 @@ Vocal.defaultProps = {
 	lang: 'en-US',
 	grammars: null,
 	timeout: 3000,
-	ariaLabel: 'speech',
-	tabIndex: -1,
+	ariaLabel: 'microphone icon',
+	tabIndex: 0,
```

Comment:
Ne pas mettre, sauf si désactivé, alors mettre -1.

Reply:
Je peux te retrouver une source pour ça si tu veux.

Reply:
According to MDN:

`tabindex="0" means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order.`

That's what I expect.

Reply:
The point is that Vocal's default render is a button. Buttons are already focusable by default so you don't need to use tabindex to make them focusable. So you don't need to set a default tabindex. In the rare case where your component is used in a page that uses tabindexes elsewhere, it would change the tabbing order of Vocal in a way that isn't documented to your user. That's why not setting it unless you need it is preferable.

Reply:
all things considered I think you're right, delete it